lib/internal/cluster/primary.js

```diff
@@ -158,6 +159,10 @@ function removeHandlesForWorker(worker) {
   });
 }
 
+cluster.getSettings = function() {
+  return { ...cluster.settings };
```

Comment:
Isn't `cluster.settings` already an object literal? Why using spread on it?

Reply:
To copy it

Reply:
And node.js does to offer helpers to shallow and/or deep clone an object to make it explicit?

Reply:
AFAIK using the spread is the idiomatic way of copying an object in JavaScript.